Added ability to specify data to exclude from an export.

spoik opened the pull request.

Commit:
Added ability to specify data to exclude from an export.

[ticket: NO TICKET]

bvangennep reviewed (changes requested):
Good idea!, should partially solve #50
However there is a failing test, please fix this.
Also make sure the code is psr-2 compliant and add some info to the readme how to use this feature.

Commit:
Fixed PSR-2 code style violations.

[ticket: X]

Commit:
Added a section to the README that explains how to exclude data
types when exporting.

[ticket: NO TICKET]

Commit:
Fixed failing unit test.

[ticket: NO TICKET]

spoik commented:
I updated the code to to be PSR-2 compliant, updated the README outlining how to use this, and the Unit test should be fixed now. I'll keep an eye out and make sure Travis runs successfully.

spoik commented:
Woo hoo! They passed. Let me know if you think anything else should be changed.

boboldehampsink commented on README.md:
````diff
@@ -49,6 +49,35 @@ An export can be created by running:
+./vendor/bin/schematic export --exclude=assetSources
+```
+
+Multitple exclusions can also be specified:
````
Fix spelling error

boboldehampsink reviewed (changes requested):
Can you also add a test where this specific feature is tested?

Commit:
Created unit test to test excluding certain data types on export.

[ticket: NO TICKET]

Commit:
Fixed typo in the readme.

[ticket: X]

spoik commented:
Fixed the typo in the readme and added a unit test for the excludes.
